lib/geordi/db_cleaner.rb

```diff
@@ -223,7 +223,11 @@ def is_whitelisted?(dbtype, database_name)
       else
         whitelist_content = Array.new
       end
-      whitelist_content.include? database_name.sub(@derivative_dbname, '')
+      # Allow explicit whitelisting of derivative databases like projectname_test2
+      return true if whitelist_content.include? database_name
+      # whitelisting `projectname` also whitelists `projectname_test\d+`, `projectname_development
+      return true if whitelist_content.include? database_name.sub(@derivative_dbname, '')
+      return false
```

Comment:
Vielleicht könntest du das in eine `if`-`elsif`-`else` Bedingung packen, das ist glaube ich gewöhnlicher an einem Methoden Rumpf:

```suggestion
# Allow explicit whitelisting of derivative databases like projectname_test2
if whitelist_content.include? database_name
  true
# whitelisting `projectname` also whitelists `projectname_test\d+`, `projectname_development  
elsif whitelist_content.include? database_name.sub(@derivative_dbname, '')  
  true
else  
  false
end  
```